src/i18n/routing.ts

```diff
@@ -0,0 +1,13 @@
+import { createNavigation } from "next-intl/navigation";
+import { defineRouting } from "next-intl/routing";
+
+export type Locale = (typeof locales)[number];
+const locales = ["en", "no"] as const;
+
+export const routing = defineRouting({
+  locales,
+  defaultLocale: "no",
```

Comment:
I think the default could be in Norwegian for now, but we might want to consider switching to English later